### PR TITLE
Create items

### PR DESCRIPTION
**Pull request by bannastre**

### CODE REVIEW

File: controllers/listController.js [hunk 1]
```diff
@@ -13,10 +14,18 @@ exports.createList = async (req, res) => {
 
 exports.getLists = async (req, res) => {
   const lists = await List.find();
+  // const items = await Item.find({});
```
Comment (marioribeiro) [inline controllers/listController.js]: Is this needed?

**Review by marioribeiro — CHANGES_REQUESTED** [review]
Check notes

File: controllers/listController.js (hunk 1 above)
Comment (bannastre) [reply controllers/listController.js]: no it is not

**Comment by marioribeiro** [comment]
:shipit:

**Review by marioribeiro — APPROVED** [review]

**Pull request merged**